packages/main/src/MenuItem.ts

```diff
@@ -117,6 +117,16 @@ class MenuItem extends UI5Element {
 	@property()
 	accessibleName!: string;
 
+	/**
+	 * Defines the text of the tooltip that would be displayed for the menu item.
```

Comment:
I suggest the shorter:
Defines the text of the tooltip for the menu item.

Reply:
done